client/components/dashboard/QuickActionSection.js

```diff
@@ -27,10 +28,11 @@ const QuickActionsSection = () => {
    * @param {string} action - The selected action.
    */
   const handleActionSelect = (action) => {
+    const isWeb = Platform.OS == 'web';
     if (action === 'createPack') {
-      router.push('/pack/create');
+      router.push(isWeb ? '/pack/create' : '/tabs/pack/create');
```

Comment:
i'm a bit worried about logic like this. we should decide on one way to go here. i think we can go all in on tabs

Reply:
but we don't want the route to be /tabs/* in the URL